### PR TITLE
[fix]: Documentation for the Link Component

### PR DESCRIPTION
**Pull request by AsifQamar**
updated the .md file

### CODE REVIEW

**Comment by RudraPratik30** [comment]
Hacktoberfest Link Component issue [#10969](https://github.com/ToolJet/ToolJet/issues/10969)

File: docs/docs/widgets/link.md [hunk 1]
```diff
@@ -47,37 +46,37 @@ Following actions of link component can be controlled using the component specif
...
 |:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}` |
+| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}` |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
```
Comment (RudraPratik30) [inline docs/docs/widgets/link.md:80]: Hey @AsifQamar you need to add a divider before the style section.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @AsifQamar you were supposed to make changes in two files that are mentioned in the issue, please replicate the changes in the versioned docs as well, thanks!

File: docs/docs/widgets/link.md (hunk 1 above)
Comment (AsifQamar) [reply docs/docs/widgets/link.md:80]: > Hey @AsifQamar you need to add a divider before the style section.

OK

**Comment by PriteshKiri** [comment]
Hey @AsifQamar,

Can you please make the changes asked by @RudraPratik30?

Please let us know if you need any assitance.

**Comment by AsifQamar** [comment]
> Hey @AsifQamar,
> 
> Can you please make the changes asked by @RudraPratik30?
> 
> Please let us know if you need any assitance.

How to add divider, I'm not getting it

**Comment by RudraPratik30** [comment]
Hey @AsifQamar, please use [----] to add a divider, without [ ]

**Comment by AsifQamar** [comment]
> ## Hey @AsifQamar, please use [----] to add a divider, without [ ]

Ok

**Comment by RudraPratik30** [comment]
> Hey @AsifQamar you were supposed to make changes in two files that are mentioned in the issue, please replicate the changes in the versioned docs as well, thanks!

Hey @AsifQamar, please work on this as well.

**Comment by AsifQamar** [comment]
> > Hey @AsifQamar you were supposed to make changes in two files that are mentioned in the issue, please replicate the changes in the versioned docs as well, thanks!
> 
> Hey @AsifQamar, please work on this as well.

Changed both file check pr section

**Comment by RudraPratik30** [comment]
Hey @AsifQamar you are supposed to make changes in both files in a single PR.
You currently have 3 open PRs: https://github.com/ToolJet/ToolJet/pull/10983, https://github.com/ToolJet/ToolJet/pull/11009, https://github.com/ToolJet/ToolJet/pull/11008
Please make all the necessary changes in a single PR and let me know which PR I have to review. And please close the rest of the PRs.

**Comment by AsifQamar** [comment]
> Hey @AsifQamar you are supposed to make changes in both files in a single PR. You currently have 3 open PRs: #10983, #11009, #11008 Please make all the necessary changes in a single PR and let me know which PR I have to review. And please close the rest of the PRs.

refer to  #11016

**Pull request closed without merging**